api/src/main/java/ai/djl/Device.java

```diff
@@ -36,6 +36,7 @@ public final class Device {
 
     private static final Device CPU = new Device(Type.CPU, -1);
     private static final Device GPU = Device.of(Type.GPU, 0);
+    private static final Device MPS = Device.of(Type.MPS, -1);
```

Comment:
MPS is PyTorch specific, for the time being, it's better keep it PyTorch only. We can add to `Device` class when it become standard.

Reply:
I am not too sure how to avoid this part "cleanly" without having to rewrite a bunch of code in pytorche's JniUtils.java, as most of the functions there use a `Device` object to infer the PtDeviceType:
https://github.com/deepjavalibrary/djl/blob/8fce4ebf0eceda51abbce6448ca029fb5599899c/engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/JniUtils.java#L191-L202

One way is to change the `PtDeviceType:: toDeviceType(Device device)` to somehow return the code for MPS device on osx-aarm64 systems, but I don't know how to do that:
https://github.com/deepjavalibrary/djl/blob/8fce4ebf0eceda51abbce6448ca029fb5599899c/engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtDeviceType.java#L28-L38